lua/leetcode/config/template.lua

```diff
@@ -110,6 +110,11 @@ local M = {
         focus_testcases = "H", ---@type string
         focus_result = "L", ---@type string
     },
+
+    inject = {
+      before = "", ---@type string|string[]
+      after = "", ---@type string|string[]
+    }
```

Comment:
I would change it to have per language configuration

```
 injector = {} ---@type table<lc.lang, lc.inject>
```

and then in the README you can add something like this as an example

```
injector = {
      ["java"] =  {
          before = "import java.util.*;", ---@type string|string[]
          after = "", ---@type string|string[]
     }
}
```

at the top of this file you can add the missing alias

```
---@alias lc.inject { before?: string|string[], after?: string|string[] } 
```